src/domains/messages-app/11_function_cqrs.tf

```diff
@@ -1,3 +1,55 @@
+resource "azurerm_resource_group" "backend_messages_rg" {
+  name     = format("%s-backend-messages-rg", local.product)
+  location = var.location
+
+  tags = var.tags
+}
+
+data "azurerm_eventhub_authorization_rule" "evh_ns_io_auth_messages" {
+  name                = "${var.prefix}-messages"
+  namespace_name      = "${local.product}-evh-ns"
+  eventhub_name       = "${var.prefix}-cosmosdb-message-status-for-view"
+  resource_group_name = "${local.product}-evt-rg"
+}
+
+data "azurerm_eventhub_authorization_rule" "evh_ns_io_auth_cdc" {
+  name                = "${var.prefix}-cdc"
+  namespace_name      = "${local.product}-evh-ns"
+  eventhub_name       = "${var.prefix}-cosmosdb-message-status-for-view"
+  resource_group_name = "${local.product}-evt-rg"
+}
+
+data "azurerm_eventhub_namespace" "messages-weu-prod01" {
+  name                = "${local.product}-messages-weu-prod01-evh-ns"
+  resource_group_name = "${local.product}-messages-weu-prod01-evt-rg"
+}
+
+data "azurerm_eventhub_authorization_rule" "io-p-payments-weu-prod01-evh-ns_payment-updates_io-fn-messages-cqrs" {
+  name                = "${var.prefix}-fn-messages-cqrs"
```

Comment:
can't we get these value from output or module instead of data?

Reply:
done, where possible